spec/javascripts/components/option-select-spec.js

```diff
@@ -73,382 +74,383 @@ describe('An option select component', function() {
+    })
 
     it('calls optionSelect.open() if the optionSelect is currently closed', function () {
-      $element.addClass('js-closed');
-      spyOn(optionSelect, 'open');
-      optionSelect.toggleOptionSelect(jQuery.Event('click'));
-      expect(optionSelect.open.calls.count()).toBe(1);
-    });
-  });
+      $element.addClass('js-closed')
+      spyOn(optionSelect, 'open')
+      optionSelect.toggleOptionSelect(jQuery.Event('click'))
+      expect(optionSelect.open.calls.count()).toBe(1)
+    })
+  })
 
   describe('when the open/close button is clicked', function () {
     beforeEach(function () {
-      $element = $(html);
-      $('body').append($element);
-      optionSelect.start($element);
-    });
+      $element = $(html)
+      $('body').append($element)
+      optionSelect.start($element)
+    })
 
     it('closes and opens the option select', function () {
-      var $button = $element.find('button');
-      $button.click();
-      expect($element.hasClass('js-closed')).toBe(true);
+      var $button = $element.find('button')
+      $button.click()
+      expect($element.hasClass('js-closed')).toBe(true)
 
-      $button.click();
-      expect($element.hasClass('js-closed')).toBe(false);
-    });
+      $button.click()
+      expect($element.hasClass('js-closed')).toBe(false)
+    })
 
     it('updates aria-expanded accordingly', function () {
-      var $button = $element.find('button');
-      $button.click();
-      expect($element.find('button').attr('aria-expanded')).toBe('false');
+      var $button = $element.find('button')
+      $button.click()
+      expect($element.find('button').attr('aria-expanded')).toBe('false')
 
-      $button.click();
-      expect($element.find('button').attr('aria-expanded')).toBe('true');
-    });
-  });
+      $button.click()
+      expect($element.find('button').attr('aria-expanded')).toBe('true')
+    })
+  })
 
-  describe ('isCheckboxVisible', function () {
-    var firstCheckbox, lastCheckbox;
+  describe('isCheckboxVisible', function () {
+    var firstCheckbox, lastCheckbox
 
     beforeEach(function () {
-      $element = $(html);
-      $('body').append($element);
-      optionSelect.start($element);
+      $element = $(html)
+      $('body').append($element)
+      optionSelect.start($element)
 
-      optionSelect.setContainerHeight(100);
-      optionSelect.$optionsContainer.width(100);
-      firstCheckbox = optionSelect.$allCheckboxes[0];
-      lastCheckbox = optionSelect.$allCheckboxes[optionSelect.$allCheckboxes.length -1];
-    });
+      optionSelect.setContainerHeight(100)
+      optionSelect.$optionsContainer.width(100)
+      firstCheckbox = optionSelect.$allCheckboxes[0]
+      lastCheckbox = optionSelect.$allCheckboxes[optionSelect.$allCheckboxes.length - 1]
+    })
 
     it('returns true if a label is visible', function () {
-      expect(optionSelect.isCheckboxVisible.call(optionSelect, 0, firstCheckbox)).toBe(true);
```

Comment:
Can you explain this change please?

Reply:
StandardJS complained about unnecessary usage of `.call` (if the first parameter of `.call` is the same as the object that is calling `.call`, then it's effectively the same as if the object calls the method directly.

i.e.

```js
foo.call(foo, bar, 1)
```

is the same as

```js
foo.bar(1)
```